Catalog/Shared/Progress/ProgressStyles.swift

```diff
@@ -69,6 +70,19 @@ struct DottedProgressStyleSample: PropSampleable {
     }
 }
 
+struct BarsProgressStyleSample: PropSampleable {
+    let name = "Bars progress view"
+    let notes: String? = ".progressViewStyle(BarsProgressStyle())"
+    let source = "Props"
+    let keywords = "progress determinate custom bars"
+    @ViewBuilder var body: AnyView {
+        AnyView(
+            ProgressExample(total: 100)
+                .progressViewStyle(BarsProgressStyle())
```

Comment:
```suggestion
                .progressViewStyle(.bars)
```
Could you add the enum style static var, please?